README.adoc

```diff
@@ -22,7 +22,7 @@ repositories {
 }
 
```

Comment:
let's omit updating README.adoc until the release is published to Maven central.